runtime-react/pom.xml

```diff
@@ -0,0 +1,197 @@
+                                        <exclude>**/codestart.yml</exclude>
+                                    </excludes>
+                                    <filtering>false</filtering>
+                                </resource>
+                            </resources>
+                            <outputDirectory>${project.build.directory}/filtered-codestarts/codestarts</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>generate-codestart-jar</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                        <configuration>
+                            <classesDirectory>${project.build.directory}/filtered-codestarts</classesDirectory>
+                            <includes>
+                                <include>codestarts/**</include>
+                            </includes>
+                            <excludes>
+                                <exclude>codestarts/**/hilla-scaffold-package-name-holder.txt</exclude>
+                            </excludes>
+                            <classifier>codestarts</classifier>
+                            <skipIfEmpty>true</skipIfEmpty>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+    <profiles>
+        <profile>
+            <id>update-hilla-codestart</id>
+            <build>
+                <defaultGoal>org.codehaus.mojo:build-helper-maven-plugin:bsh-property@hilla-scaffold</defaultGoal>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>build-helper-maven-plugin</artifactId>
+                        <version>3.4.0</version>
+                        <executions>
+                            <execution>
+                                <id>hilla-scaffold</id>
+                                <goals>
+                                    <goal>bsh-property</goal>
+                                </goals>
+                                <configuration>
+                                    <source><![CDATA[
+import java.nio.file.CopyOption;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.nio.file.attribute.FileAttribute;
+import java.util.List;
+import dev.hilla.plugin.base.HillaAppInitUtility;
+
+codeStartFolder = project.getBasedir().toPath().resolve(
+    Path.of("src", new String[] { "main", "codestarts", "quarkus", "quakus-hilla-maven-codestart", "base" })
```

Comment:
not quarkus-hilla-react?

Reply:
It is just a local folder, that is not present in the final jar, so name does not matter that much.